styles/globals.css

```diff
@@ -0,0 +1,69 @@
+/* Canada.ca Typography */
+/* https://design.canada.ca/styles/typography.html */
+
+/* Font Families */
+h1, h2, h3, h4, h5, h6 {
+  font-family: 'Lato', 'system-ui', 'sans-serif';
```

Comment:
We could say `bold` here and then not everywhere individually below.

Reply:
Yes of course 🤦

Reply:
Fixed 👍